utilities/constants.py

```diff
@@ -42,10 +42,25 @@ class Protocols:
     HTTP: str = "http"
     HTTPS: str = "https"
     GRPC: str = "grpc"
+    REST: str = "rest"
+
+
+class HTTPRequest:
+    # You will need to `.format(token='foo')`
+    # See e.g. tests/model_registry/utils.py#L51
```

Comment:
please remove as we do not want to maintain this if lines change; i'd expect a user to understand how to use with your given example

you may want to consider rephrasing the 1st line to 
```
use string formatting to set the token value
```